samples/cpp/tutorial_code/ml/neural_network/neural_network.cpp

```diff
@@ -0,0 +1,58 @@
+#include <opencv2/ml/ml.hpp>
+
+using namespace std;
+using namespace cv;
+using namespace cv::ml;
+
+int main(int argc, char* argv[])
```

Comment:
Since you are not using the arguments passes to the sample, simply replace this line by

`int main()`